docs/src/components/HeaderBox.tsx

```diff
@@ -0,0 +1,24 @@
+import styled, { css } from "styled-components";
+import { mediaQueries as mq } from "@kiwicom/orbit-components";
+
+import { MAX_CONTENT_WIDTH } from "../consts";
+
+const HeaderBox = styled.div`
+  ${mq.desktop(css`
+    position: relative;
+    height: 100vh;
+    padding: 64px;
+    background: linear-gradient(80.68deg, #f1f4f7 7.05%, #f5f7f9 57.05%);
+    z-index: 0;
+  `)}
+
+  display: block;
+  border-radius: 24px;
+  border-bottom-right-radius: 50%;
```

Comment:
could you explain that? I just don't get why it has to be 50%, it's visible only while the image is not loaded

Reply:
Yeah, I didn't think of that. It was set to 50% because of the image positioning not matching the border radius, but now I just noticed that it actually matches perfectly, so I'll remove it.